kfs-core/src/main/java/org/kuali/kfs/fp/businessobject/ProcurementCardDefault.java

```diff
@@ -19,13 +19,19 @@
 package org.kuali.kfs.fp.businessobject;
```

Comment:
I've created the same class in arizona.edu with just the missing fields, reconcilerGroupName is gone and instead of cardApprovalOfficial there is creditCardLastFour